### ISSUE TITLE
single-file-example.html link broken

### ISSUE THREAD

**Issue by WxNzEMof** [reactjs/react.dev#6832]
The page https://legacy.reactjs.org/docs/getting-started.html#try-react has a link with the text download this HTML file", that currently goes to https://raw.githubusercontent.com/reactjs/reactjs.org/main/static/html/single-file-example.html. That URL is a 404.

**Comment by Rekl0w** [comment]
![image](https://github.com/reactjs/react.dev/assets/91488737/36d3cbe0-2e96-4d37-99a0-b9792c3f1ee3)

There is a warning about that. This docs are old.

**Comment by WxNzEMof** [comment]
Ah, I did not notice that.

That page is the number one result for [the Google query "react getting started"](https://www.google.com/search?q=react+getting+started).

Maybe worth adding some meta tags to remove or de-prioritize it from search engines?

**Comment by Rekl0w** [comment]
I submitted a PR about that.

**Comment by WxNzEMof** [comment]
Thanks! Though, I think `rel="canonical"` or a `robots.txt` entry might be more effective.

**Comment by WxNzEMof** [comment]
> That page is the number one result for [the Google query "react getting started"](https://www.google.com/search?q=react+getting+started).

Another one: https://www.google.com/search?q=useeffect

**Comment by rickhanlonii** [comment]
Yeah we gotta fix these SEO issues, thanks everyone for flagging and making suggestions. I'm swamped with React Conf but can help review and land fixes afterwards.
